### PR TITLE
🚮 Remove experiment `amp-consent-granular-consent`

### PR DESCRIPTION
**Pull request by alanorozco**
Cleanup for #31607

### CODE REVIEW

File: extensions/amp-consent/0.1/test/test-amp-consent.js [hunk 1]
```diff
@@ -1222,11 +1164,11 @@ describes.realWin(
...
             },
           };
-          toggleExperiment(win, 'amp-consent-granular-consent', true);
         });
 
         afterEach(() => {
-          toggleExperiment(win, 'amp-consent-granular-consent', false);
+          /* toggleExperiment(win, 'amp-consent-granular-consent', false) // launched: true */
+          false;
         });
```
Comment (dmanek) [inline extensions/amp-consent/0.1/test/test-amp-consent.js:1172]: This block can be deleted.

File: extensions/amp-consent/0.1/test/test-amp-consent.js [hunk 2]
```diff
@@ -1583,7 +1524,8 @@ describes.realWin(
       });
 
       afterEach(() => {
-        toggleExperiment(win, 'amp-consent-granular-consent', false);
+        /* toggleExperiment(win, 'amp-consent-granular-consent', false) // launched: true */
+        false;
       });
```
Comment (dmanek) [inline extensions/amp-consent/0.1/test/test-amp-consent.js:1529]: Delete this too.

File: extensions/amp-consent/0.1/test/test-consent-ui.js [hunk 3]
```diff
@@ -298,86 +297,46 @@ describes.realWin(
...
-      it('should expand the promptUISrc', function* () {
+      it('should load the iframe, then show it with correct state CSS classes', async () => {
         const config = dict({
-          'promptUISrc': 'https://example.test?cid=CLIENT_ID&r=RANDOM',
-          'clientConfig': {
-            'test': 'ABC',
-          },
+          'promptUISrc': 'https://promptUISrc',
         });
         consentUI = new ConsentUI(mockInstance, config);
```
Comment (dmanek) [inline extensions/amp-consent/0.1/test/test-consent-ui.js:304]: nit: consentUi (lowercase trailing "i")

**Review by dmanek — APPROVED** [review]

File: extensions/amp-consent/0.1/test/test-amp-consent.js (hunk 1 above)
Comment (alanorozco) [reply extensions/amp-consent/0.1/test/test-amp-consent.js]: Good catch.

**Review by kristoferbaxter — APPROVED** [review]

**Review by caroqliu — APPROVED** [review]
Does this PR also close #31607?

**Comment by alanorozco** [comment]
@caroqliu The tracker includes an item for E2E tests that have not been added. It's unclear to me whether we need them still.

**Pull request merged**